app/forms/returns/credits/ConvertedCreditsFormProvider.scala

```diff
@@ -20,10 +20,15 @@ import javax.inject.Inject
 import forms.mappings.Mappings
 import play.api.data.Form
 import play.api.data.Forms.mapping
+import play.api.libs.json.{Json, OFormat}
 import uk.gov.voa.play.form.ConditionalMappings.{isEqual, mandatoryIf}
 
 case class ConvertedCreditsAnswer(yesNo: Boolean, weight: Option[Long])
```

Comment:
as this is now used in 2 places (form & cache) can this be moved to models? Check with carl I also suggested this on his pr he may have done it, and cause conflicts